src/modules/contacts/pages/ContactOverviewPage.vue

```diff
@@ -17,34 +17,196 @@
     </div>
     <div>
       <AppHeading>{{ t('contactOverview.information') }}</AppHeading>
+      <AppInfoList>
+        <AppInfoListItem
+          :name="'Preferred channel'"
+          :value="contact.profile.preferredContact"
+        />
+        <AppInfoListItem
+          :name="t('contacts.data.email')"
+          :value="contact.email"
+        />
+        <AppInfoListItem
+          :name="t('contacts.data.phone')"
+          :value="contact.profile.telephone"
+        />
+        <AppInfoListItem
+          :name="t('contacts.data.deliveryOptIn')"
+          :value="contact.profile.deliveryOptIn"
+        />
+        <AppInfoListItem
+          :name="t('contacts.data.newsletter')"
+          :value="contact.profile.newsletterStatus"
+        />
+        <AppInfoListItem
+          :name="t('contacts.data.groups')"
+          :value="contact.profile.newsletterGroups.join(',')"
+        />
+      </AppInfoList>
+    </div>
+
+    <div>
+      <AppHeading>{{ t('contactOverview.contribution') }}</AppHeading>
+      <AppInfoList>
+        <AppInfoListItem
+          :name="t('contacts.data.amount')"
+          :value="
+            contact.contributionAmount
+              ? n(contact.contributionAmount, 'currency')
+              : '-'
+          "
+        />
+        <AppInfoListItem
+          :name="t('contacts.data.period')"
+          :value="contact.contributionPeriod"
+        />
+        <AppInfoListItem
+          v-if="contact.contribution.type === ContributionType.GoCardless"
+          :name="t('contacts.data.payingFee')"
+          :value="contact.contribution.payFee"
```

Comment:
This should be internationalised using common.yes and common.no

Reply:
Done: https://github.com/beabee-communityrm/beabee-frontend/pull/104/commits/578f343a4e4679ed35823e36a989aa613b9789b8